v0.4/platform/platform/src/__tests__/i18n.test.ts

```diff
@@ -38,21 +40,47 @@ describe('i18n', () => {
     expect(translated).toBe('Loading plugin <b>xxx</b>...')
   })
 
-  it('should return status when no loader', async () => {
-    const translated = translate('component.id' as IntlString, {})
-    return await expect(translated).rejects.toThrowError("ERROR: platform.NoLoaderForStrings")
+  it('should emmit status when no loader', async () => {
+    const component = 'component'
+    const message = `${component}.id`
+
+    const checkStatus = new Status(Severity.ERROR, Code.NoLoaderForStrings, { component })
+    const eventListener = async (event: string, data: any): Promise<void> => {
+      await expect(data).toEqual(checkStatus)
+    }
+    addEventListener(PlatformEvent, eventListener)
+    await translate(message as IntlString, {})
```

Comment:
You need separate test to check if `translate` returns expected value or check it in this test as well. Also `emit` word has one 'm'.

Reply:
done as separate etst